astroquery/gaia/tests/test_gaiatap.py

```diff
@@ -14,6 +14,7 @@
 
 
 """
+import unittest
```

Comment:
The use of `unittest` was just removed by 6ec71cecceef295c78408aaa056eda706e467ea1, it should not be reintroduced without a very good reason.